P5/Source/Specs/sex.xml

```diff
@@ -8,15 +8,15 @@ $Id$
 -->
 <?xml-model href="https://jenkins.tei-c.org/job/TEIP5-dev/lastSuccessfulBuild/artifact/P5/release/xml/tei/odd/p5.nvdl" type="application/xml" schematypens="http://purl.oclc.org/dsdl/nvdl/ns/structure/1.0"?>
 <elementSpec xmlns="http://www.tei-c.org/ns/1.0" module="namesdates" ident="sex">
-  <gloss xml:lang="en" versionDate="2009-03-19">sex</gloss>
+  <gloss xml:lang="en" versionDate="2022-05-10">sex</gloss>
```

Comment:
Is this on purpose?

Reply:
Probably not since there is no chance to the gloss. Reverted.